lib/generators.nix

```diff
@@ -209,9 +209,17 @@ rec {
         else
           ''${section} "${subsection}"'';
 
+      mkValueString = v:
+        let
+          escapedV = ''
+            "${
+              replaceStrings [ "\n" "	" ''"'' "\\" ] [ "\\n" "\\t" ''\"'' "\\\\" ] v
```

Comment:
It's not immediately clear to me why you are replacing spaces with tabs? Can't we just leave the spaces?